src/Core/defaultValue.js

```diff
@@ -7,7 +7,7 @@
 
 import { Vector3 } from 'three';
 
-var defaultValue = function defaultValue(value, def) {
+const defaultValue = function defaultValue(value, def) {
```

Comment:
Yep, I hesitated, but decided against it, because `defaultValue` has quite a few use. I thought it would need another Pr.